docs/data/base/components/button/button.md

```diff
@@ -8,37 +8,131 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/button/
 
 # Unstyled button
 
-<p class="description">Buttons allow users to take actions and make choices with a single tap.</p>
+<p class="description">Buttons let users take actions and make choices with a single tap.</p>
 
-## Basic button
+## Introduction
 
-```js
+`ButtonUnstyled` replaces the native HTML `<button>` element.
```

Comment:
It could be interesting to explain why. Why do we need to replace it in the first place?